.github/workflows/rawls-run-azure-e2e-tests.yaml

```diff
@@ -283,8 +302,9 @@ jobs:
       - name: Notify slack
         uses: slackapi/slack-github-action@v1.23.0
         with:
-          # Channel is for #dsp-workspaces-test-alerts
-          channel-id: 'C03F21QEWV7'
-          slack-message: "Azure E2E Tests FAILED, branch: ${{ needs.init-github-context.outputs.branch }}\n${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}"
+          # C03F21QEWV7 channel is for #dsp-workspaces-test-alerts
+          # C53JYBV9A channel is for #dsde-qa
+          channel-id: ${{ inputs.bee-version-template == 'staging' && 'C03F21QEWV7,C53JYBV9A' || 'C03F21QEWV7' }}
+          slack-message: "Azure E2E Tests FAILED, branch: ${{ inputs.build-branch && needs.init-github-context.outputs.branch || inputs.bee-version-template }}\n${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}"
```

Comment:
Can see examples of this handling the `build-branch` option in our test Slack channel: https://broadinstitute.slack.com/archives/C03F21QEWV7/p1702583233773779 (did not build branch) and https://broadinstitute.slack.com/archives/C03F21QEWV7/p1702583711971539 (did build).

Reply:
However, in the staging case we probably want to always notify (like we do for the billing project test) so that Gary can see the results. I can make that change.

Reply:
I will also need to make the messaging smarter so notification doesn't assume failure.